src/resources/views/corporation/wallet/journal/journal.blade.php

```diff
@@ -55,17 +74,14 @@
         {data: 'amount', name: 'amount'},
         {data: 'balance', name: 'balance'}
       ],
-      dom: '<"row"<"col-sm-6"l><"col-sm-6"f>><"row"<"col-sm-6"i><"col-sm-6"p>>rt<"row"<"col-sm-6"i><"col-sm-6"p>><"row"<"col-sm-6"l><"col-sm-6"f>>',
-      "fnDrawCallback": function () {
-        $(document).ready(function () {
-          $("[data-toggle=tooltip]").tooltip();
-          $("img").unveil(100);
-          ids_to_names();
-        });
+      DrawCallback: function () {
```

Comment:
shouldn't be a lower `d` here ?
https://datatables.net/reference/option/drawCallback

Reply:
jop resolved via webUI